tests/test_jn.py

```diff
@@ -40,13 +40,18 @@ def collect_jn_errors(nb):
 
     return errors
 
+def embedding_fail(error_list):
+    if error_list != [] and error_list[0].evalue == 'no embedding found':
+        return True
+    return False
```

Comment:
I would rather add errors to the list with `errors.append(output.evalue)` in the errors collect function, and then use
```suggestion
    return error_list and 'no embedding found' in error_list
```

Reply:
That's more elegant for the case of an embedding failure but for all other error cases we'll get less information in CircleCI on the cause of the failure because the error cell contains additional useful keys: 
  'ename,  'evalue', &  'traceback'

Reply:
Got it. Well, in that case it's probably better to traverse the `error_list` here, instead checking only index 0.

Reply:
`robust_run_jn()` reruns if the JN failed due to embedding so it would be index 0 but if another error occurs first I want that exposed, so don't want the JN to rerun.

Reply:
In that case, I'd go with:
```
return error_list and error_list[0].evalue == 'no embedding found'
```